roles/pulp-devel/templates/venv.bashrc.j2

```diff
@@ -3,8 +3,11 @@ export WORKON_HOME={{pulp_install_dir}}/..
 export PIP_VIRTUALENV_BASE=$WORKON_HOME
 export VIRTUALENV_USE_DISTRIBUTE=true
 export PIP_RESPECT_VIRTUALENV=true
+{% if ansible_distribution == 'Fedora' %}
 # make sure python3 is used in virtual environments
 export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
 source /usr/bin/virtualenvwrapper-3.sh
-# Configure settings for django-admin
-export DJANGO_SETTINGS_MODULE=pulpcore.app.settings
```

Comment:
Why was this env variable removed?

Reply:
It is completely unrelated to venv. Also it is already in the file next to it.

Reply:
https://github.com/pulp/ansible-pulp/blob/master/roles/pulp-devel/files/django.bashrc#L1